lib/features/mailbox_dashboard/domain/usecases/get_composer_cache_on_web_interactor.dart

```diff
@@ -9,9 +11,9 @@ class GetComposerCacheOnWebInteractor {
 
   GetComposerCacheOnWebInteractor(this.composerCacheRepository);
 
-  Either<Failure, Success> execute() {
+  Either<Failure, Success> execute(AccountId accountId, UserName userName) {
```

Comment:
Should be return Stream<Either<Failure, Success>>

Reply:
Done